src/types.rs

```diff
@@ -145,9 +150,14 @@ pub enum VideoPeriod {
 }
 
 /// Type of video
-#[derive(PartialEq, serde::Deserialize, serde::Serialize, Clone, Debug)]
+#[derive(PartialEq, Eq, Deserialize, Serialize, Clone, Debug)]
 #[serde(rename_all = "snake_case")]
 pub enum VideoType {
+    /// A live video
+    Live,
+    // FIXME: What is this?
+    /// A playlist video
+    Playlist,
```

Comment:
found in https://dev.twitch.tv/docs/eventsub/eventsub-reference/#stream-online-event